app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    @abstractmethod
+    def eat_favourite_dish(self) -> None:
```

Comment:
The `eat_favourite_dish` method is currently defined to return `None`. However, the previous review mentioned it should return a `str`. Please verify the task requirements and adjust the return type if necessary.